index.html

```diff
@@ -42,10 +42,14 @@
     <button role="main" id="keyboard" class="hidden esri-widget esri-component esri-widget-button keyboard-btn esri-icon-announcement"
         aria-label="Enable keyboard navigation">
     </button>
+    <div id="esri-a11y-table-toggle" class="esri-component esri-toggle-table esri-widget--button esri-widget hidden" role="button" tabindex="0" aria-label="Toggle table" title="Toggle table">
```

Comment:
If possible switch this to use a button instead of a div with the role of button.

Reply:
Handled in 11314bd